src/dev-server.ts

```diff
@@ -89,14 +115,21 @@ class Server {
 
       const found = (
         getFile(url) ??
+        getFile(url + '.html') ??
```

Comment:
Why?

Reply:
@tcardlab I've personally never liked how https://github.com/sdegutis and https://github.com/sdegutis/ are the same page but technically two different URLs, and there's no redirect. Maybe it's just me...